klayout/drc/rule_decks/dualgate.drc

```diff
@@ -0,0 +1,87 @@
+################################################################################################
```

Comment:
can we lint/format those with https://rubocop.org/ ?

Reply:
same as the above #14